templates/alloy-deployment.yaml

```diff
@@ -25,7 +25,7 @@ spec:
            "--config",
            "/etc/alloy/config.yaml",
            "--facility-code",
-           "{{ .Values.location }}",
+           "{{ .Values.clusterInfo.facility }}",
```

Comment:
I'd prefer we don't unilaterally change things in the sandbox like this.

Reply:
Ok, I just wanted it to be more consistent with what we do in our helm charts. If I got any pushback I was going to remove it.